versions/ufo3/lib.plist.md

```diff
@@ -29,6 +29,49 @@ This defines a preferred glyph name to Postscript glyph name mapping for glyphs
 
 The mapping is stored as a dictionary with glyphs names as keys and Postscript glyph names as values. Both keys and values must be strings. The values must conform to the Postscript glyph naming specification. The dictionary may contain glyph names that are not in the font. The dictionary may not contain a key, value pair for all glyphs in the font. If a glyph's name is not defined in this mapping, the glyph's name should be used as the Postscript name.
 
+#### public.postscript.hints
+
+This key provides a mapping from glyph names to sets of PostScript hints for the
+default glyph layer. The mapping is stored as a dictionary with glyph names as
```

Comment:
Why not store the same data in the lib in the glif? That would make it much simpler and removes the possibility to loose track when the glyph names change.

Reply:
That makes sense to me. I had thought Tal was requesting that this be in lib.plist file, but I see that is not the case. I will modify the proposal to store the hint key in the GLIF file lib.